CHANGELOG.md

```diff
@@ -4,6 +4,16 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+### Changed
+- `SubstanceParameter` encodings are now computed exclusively with the
+  `scikit-fingerprints` package, granting access to all fingerprints available therein
+
+### Deprecations
+- `SubstanceEncoding` value `MORGAN_FP`. Please use `ECFP` with 1024 bits and
```

Comment:
reworded, hopefully fine. Would still keep info on replacement

Reply:
Summoning @Scienfitz here - should this be part of the CHANGELOG? I think it should not, would require a second opinion here :)

Reply:
I dont think it hurts, but if you insist for consistency reasons imo its fine to remove it

Reply:
Looks good to me 👍🏼 In fact, our other deprecation entries also usually mention what is the corresponding replacement. So I'll resolve since I see text has already been updated